Sources/TokamakCore/Fiber/Layout/StackLayout.swift

```diff
@@ -0,0 +1,288 @@
+      totalSpacing += spacing
+
+      // If this `View` is the widest, save it to the cache for access in `placeSubviews`.
+      if min.size[keyPath: Self.crossAxis] > cache.maxSubview?.size[keyPath: Self.crossAxis]
+        ?? .zero
+      {
+        cache.maxSubview = min
+      }
+
+      return MeasuredSubview(
+        view: view,
+        index: index,
+        min: min.size,
+        max: max,
+        infiniteMainAxis: max[keyPath: Self.mainAxis] == .infinity,
+        spacing: spacing
+      )
+    }
+
+    // Calculate ideal sizes for each View based on their min/max sizes and the space available.
+    var available = proposal[keyPath: Self.mainAxis] - minSize - totalSpacing
+    /// The final resulting size.
+    var size = CGSize.zero
+    size[keyPath: Self.crossAxis] = cache.maxSubview?.size[keyPath: Self.crossAxis] ?? .zero
+    for subview in measuredSubviews.sorted(by: {
+      // Sort by priority descending.
+      if $0.view.priority == $1.view.priority {
+        // If the priorities match, allow non-flexible `View`s to size first.
+        return $1.infiniteMainAxis && !$0.infiniteMainAxis
+      } else {
+        return $0.view.priority > $1.view.priority
+      }
+    }) {
+      /// The amount of space available to `View`s with this priority value.
+      let priorityAvailable = available + prioritySize[subview.view.priority, default: 0]
+      /// The number of `View`s with this priority value remaining as a `CGFloat`.
+      let priorityRemaining = CGFloat(priorityCount[subview.view.priority, default: 1])
+      // Propose the full `crossAxis`, but only the remaining `mainAxis`.
+      // Divvy up the available space between each remaining `View` with this priority value.
+      var divviedSize = proposal
+      divviedSize[keyPath: Self.mainAxis] = priorityAvailable / priorityRemaining
+      let idealSize = subview.view.sizeThatFits(.init(divviedSize))
+      cache.idealSizes[subview.index] = idealSize
+      size[keyPath: Self.mainAxis] += idealSize[keyPath: Self.mainAxis] + subview.spacing
+      // Remove our `idealSize` from the `available` space.
+      available -= idealSize[keyPath: Self.mainAxis]
+      // Decrement the number of `View`s left with this priority so space can be evenly divided
+      // between the remaining `View`s.
+      priorityCount[subview.view.priority, default: 1] -= 1
+    }
+    return size
+  }
+
+  func spacing(subviews: Subviews, cache: inout Cache) -> ViewSpacing {
+    subviews.reduce(into: .zero) { $0.formUnion($1.spacing) }
+  }
+
+  func placeSubviews(
+    in bounds: CGRect,
+    proposal: ProposedViewSize,
+    subviews: Subviews,
+    cache: inout Cache
+  ) {
+    /// The current progress along the `mainAxis`.
+    var position = CGFloat.zero
+    /// The offset of the `_alignment` in the `maxSubview`,
+    /// used as the reference point for alignments along this axis.
+    let alignmentOffset = cache.maxSubview?[alignment: _alignment, in: Self.orientation] ?? .zero
```

Comment:
nit: `///` on local declarations won't have any effect
```suggestion
    // The current progress along the `mainAxis`.
    var position = CGFloat.zero
    // The offset of the `_alignment` in the `maxSubview`,
    // used as the reference point for alignments along this axis.
    let alignmentOffset = cache.maxSubview?[alignment: _alignment, in: Self.orientation] ?? .zero
```